test/recipes/80-test_pkcs12.t

```diff
@@ -97,7 +97,7 @@ SKIP: {
 
 SKIP: {
     skip "Skipping legacy PKCS#12 test because RC2 is disabled in this build", 1
-        if disabled("rc2") || disabled("legacy");
+        if disabled("des") || disabled("rc2") || disabled("legacy");
```

Comment:
What about adjusting the skip info to: `Skipping legacy PKCS#12 test because required algorithms are disabled`

Reply:
done